test/e2e/framework/volume/fixtures.go

```diff
@@ -155,8 +156,11 @@ func NewNFSServer(cs clientset.Interface, namespace string, args []string) (conf
 	if len(args) > 0 {
 		config.ServerArgs = args
 	}
-	pod, ip = CreateStorageServer(cs, config)
-	return config, pod, ip
+	pod, host = CreateStorageServer(cs, config)
+	if strings.Contains(host, ":") {
+		host = "[" + host + "]"
```

Comment:
Is this format exclusive to NFS or should this be performed inside CreateStorageServer()?

Reply:
generally, when you have to pass a host:port, the IPv6 is enclosed with square brackets because it contains `:`.
It is possible that if all other storage servers use the host:port format will need the square brackets, let me check this

Reply:
I'm not feeling confident generalizing, mainly because there is no testing with IPv6 for the other backends.
We are going to test NFS with IPv6 with KIND, and that's why we found this issue.
I suggest to do the changes per provider, so we can be sure we are not breaking anything.

Reply:
Fair enough.  I figured it was worth asking just to be sure.